awscli/examples/s3/sync.rst

```diff
@@ -81,4 +81,4 @@ Output::
 
 The following ``sync`` command syncs files between two buckets in different regions::
 
-    aws s3 sync s3://my-us-west-2-bucket s3://my-us-east-1-bucket --source-region us-west-2 --region us-east-1
\ No newline at end of file
+    aws s3 sync s3://my-us-west-2-bucket s3://my-us-east-1-bucket --source-region us-west-2 --region us-east-1
```

Comment:
Now that you note it, I'm equally curious. I only made the change to line 71, in the github editor, and clearly didn't review exactly what changes it thought occurred.

It looks like the final line, so I'm guessing that I completely messed up the last line by changing whether the file had an empty line at the end.

I'm not familiar with the best way to fix this. Do I make the correct change then make a new pull request?